src/libs/ReportUtils.ts

```diff
@@ -5534,6 +5540,24 @@ function getDeletedTransactionMessage(action: ReportAction) {
     return message;
 }
 
+function getMovedTransactionMessage(action: ReportAction) {
+    const movedTransactionOriginalMessage = getOriginalMessage(action as ReportAction<typeof CONST.REPORT.ACTIONS.TYPE.MOVED_TRANSACTION>) ?? {};
+    const {toReportID} = movedTransactionOriginalMessage as OriginalMessageMovedTransaction;
+    const message = translateLocal('iou.movedTransaction', {
+        reportID: toReportID,
```

Comment:
We should pass the url to the report here so we can link it, as well as the reportName